### PR TITLE
Document new vertical navigation

### PR DESCRIPTION
**Pull request by adamlazik1**
Foreman 3.9 introduces submenus, which needs to be reflected in the documentation.

Review commit by commit for easier review. 

I included the commands used in commit messages.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by maximiliankolb** [comment]
Thanks Adam. I will have a look at this until Monday.

**Comment by mjivraja** [comment]
Despite a large number of files, this looks mostly good to me. Will again take a look to share my final view.

**Comment by maximiliankolb** [comment]
# How I reviewed this PR locally

idea: review by reproducing the change & reviewing the `sed` commands

1. git fetch upstream pull/2581/head:review_2581 # source: https://stackoverflow.com/a/30584951
2. git switch -c reproduce_2581 # create your own local branch based on "master"
3. gitk review_2581 &

* review the `sed` commands; oldest first; checked on Foreman 3.8.0 > @adamlazik1 I assume this means that you can chery-pick all commits to "3.8"
* run the sed commands on my own branch
* create a single commit after running _all_ sed commands
* review the last commit by hand and cherry-pick it
* compare Adam's branch to mine; expect a diff because we've merge in "master" in the meantime
* rebase Adam's branch and compare it again; expecting no diff :tada: 

# Results

* for commit "content types": you're missing Deb Packages:
image
* for commit "templates": where does "Sync Templates" come from? https://github.com/theforeman/foreman_templates?
* for commit "compliance": could not verify in Foreman WebUI due to missing foreman_openscap plug-in
* for commit "Ansible": we should also do this for Puppet and Salt; it's the same idea:
image
* commit "RHCloud": could not verify in Foreman WebUI due to missing plug-in
* commit "Webhook": could not verify in Foreman WebUI due to missing plug-in

I assume Malhar has checked on Foreman with all mentioned plug-ins :+1:

**Review by maximiliankolb — APPROVED** [review]

**Comment by adamlazik1** [comment]
> for commit "content types": you're missing Deb Packages:

I did miss this because it is not in Satellite, however, the search for "Deb Packages" returns empty:
```
$ git grep 'Deb Packages'
```
> for commit "templates": where does "Sync Templates" come from? https://github.com/theforeman/foreman_templates?

I guess it is only in Satellite:
image

> for commit "compliance": could not verify in Foreman WebUI due to missing foreman_openscap plug-in

image

> for commit "Ansible": we should also do this for Puppet and Salt; it's the same idea:

Thanks, I missed those because they are not on Satellite by default. Added commits.

> commit "RHCloud": could not verify in Foreman WebUI due to missing plug-in

image

> commit "Webhook": could not verify in Foreman WebUI due to missing plug-in

image

File: guides/common/modules/proc_registering-hosts-by-using-the-bootstrap-script.adoc [hunk 1]
```diff
@@ -36,7 +36,7 @@ To create a suitable Puppet environment to be associated with a host group, foll
 ----
 # mkdir /etc/puppetlabs/code/environments/_example_environment_
 ----
-. In the {ProjectWebUI}, navigate to *Configure* > *Environments* and click *Import environment from*.
+. In the {ProjectWebUI}, navigate to *Configure* > *Puppet ENC* > *Environments* and click *Import environment from*.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_registering-hosts-by-using-the-bootstrap-script.adoc:39]: Maybe also split this line?

File: guides/common/modules/proc_propagating-scap-content-using-puppet-deployment.adoc [hunk 2]
```diff
@@ -8,7 +8,7 @@ Using this procedure, you can promote Security Content Automation Protocol (SCAP
 For more information, see {ManagingSecurityDocURL}configuring-compliance-policy-deployment-methods_security-compliance[Configuring Compliance Policy Deployment Methods] in _{ManagingSecurityDocTitle}_.
 
 .Procedure
-. In the {ProjectWebUI}, navigate to *Configure* > *Classes* and click `foreman_scap_client`.
+. In the {ProjectWebUI}, navigate to *Configure* > *Puppet ENC* > *Classes* and click `foreman_scap_client`.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_propagating-scap-content-using-puppet-deployment.adoc:11]: Maybe also split this line?

File: guides/common/modules/proc_importing-salt-states.adoc [hunk 3]
```diff
@@ -6,7 +6,7 @@ You can import Salt States from your Salt Master to {Project}.
 The Salt Master configuration in this guide uses a Salt Environment called `base` that includes the Salt States stored in `/srv/salt/`.
 
 .Procedure
-. In the {ProjectWebUI}, navigate to *Configure* > *Salt States* and click *Import from FQDN*.
+. In the {ProjectWebUI}, navigate to *Configure* > *Salt* > *States* and click *Import from FQDN*.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_importing-salt-states.adoc:9]: Maybe also split this line?

**Review by maximiliankolb — APPROVED** [review]
Thanks for the follow-up commits! Change LGTM; maybe also split the line as in 88c7d3078d990728832c8bacd004ebd75b26f935 Your call.

**Comment by adamlazik1** [comment]
Thanks for pointing that out. I also split other lines which I previously looked over.

**Review by maximiliankolb — APPROVED** [review]
Thanks Adam, LGTM!

File: guides/common/modules/con_provisioning-templates.adoc [hunk 4]
```diff
@@ -4,9 +4,9 @@
 A provisioning template defines the way {ProjectServer} installs an operating system on a host.
 
 {ProjectName} includes many template examples.
-In the {ProjectWebUI}, navigate to *Hosts* > *Provisioning templates* to view them.
+In the {ProjectWebUI}, navigate to *Hosts* > *Templates* > *Provisioning Templates* to view them.
 You can create a template or clone a template and edit the clone.
-For help with templates, navigate to *Hosts* > *Provisioning templates* > *Create Template* > *Help*.
+For help with templates, navigate to *Hosts* > *Templates* > *Provisioning Templates* > *Create Template* > *Help*.
```
Comment (mjivraja) [inline guides/common/modules/con_provisioning-templates.adoc:9]: I went through the WebUI navigation and my perspective is a bit different.

`Templates` is the header of the group of entries, and you click one of the entries? Then I would go without adding `> Templates`. (Applies Everywhere)
If you want to help the reader find the entry in the long list, you can append `(in the Templates section)` or something similar.

I also went through the style guide but did not get any supporting section for the header to be the part of sequence of selection. However, I am open to the suggestion if mistaken.

I am okay with every other thing though.

Comment (maximiliankolb) [reply guides/common/modules/con_provisioning-templates.adoc:9]: You have to click "Hosts", then click "Templates", and then click "Provisioning Templates". To me, this means that it has to be documented as "..., navigate to *Hosts* > *Templates* > *Provisioning Templates*." So I'm 100% ACK with Adam's PR here.

Comment (mjivraja) [reply guides/common/modules/con_provisioning-templates.adoc:9]: `Templates` is not clickable in WebUI.

Comment (maximiliankolb) [reply guides/common/modules/con_provisioning-templates.adoc:9]: It _is_ clickable on Foreman 3.8.0. It does not open a page if you click it, but expand the menu.

Comment (mjivraja) [reply guides/common/modules/con_provisioning-templates.adoc:9]: Yes, my snap was not showing click-to-expand. But, confirmed with SME :+1:

**Review by mjivraja — APPROVED** [review]
Shoot! This can be cherry-picked to `3.8` as well, as per @maximiliankolb's comment.

File: guides/common/modules/con_provisioning-templates.adoc (hunk 4 above)
Comment (adamlazik1) [reply guides/common/modules/con_provisioning-templates.adoc:9]: You can also verify this by installing the latest satellite stream from ohsnap.

**Comment by adamlazik1** [comment]
Merged and cherry-picked:
   2e8dba2ac..57e926b7b  3.8 -> 3.8

**Pull request merged**